feat: add removePlugin api(resolve #3716) (#3717)
highlightjs/highlight.js: docs/api.rst

* **In Safe Mode** all other languages would continue to highlight just fine. The broken language would appear as a code block, but without any highlighting (as if it were plaintext).
* **In Debug Mode** all highlighting would stop and a JavaScript error would be thrown.

addPlugin
---------

::

  addPlugin(plugin)

> Add a plugin for hljs.See the :doc:`plugin api </plugin-api>` for more plugin information.

Reviewer: ```suggestion
Add a plugin to this instance of Highlight.js.  See the :doc:`plugin api </plugin-api>` for additional plugin information.
```

removePlugin
---------

::

  removePlugin(plugin)

> Remove a specified plugin for hljs.

Reviewer: ```suggestion
Remove the specified plugin from this instance.  ``plugin`` must be exactly the same object that was passed to 
``addPlugin``.
```

  addPlugin(plugin)

Add a plugin to this instance of Highlight.js.  See the :doc:`plugin api </plugin-api>` for additional plugin information.
removePlugin

> ---------

Reviewer: ```suggestion
Add a plugin to this instance of Highlight.js.  See the :doc:`plugin api </plugin-api>` for additional plugin information.

removePlugin
---------
```
